integration-tests/cucumber-tests-platform-publiclighting/src/test/resources/features/publiclighting/osgp-adapter-ws-publiclighting/ScheduleManagement/SetLightSchedule.feature

```diff
@@ -276,6 +336,7 @@ Feature: PublicLightingScheduleManagement Set Light Schedule
       | TriggerWindow        | <TriggerWindow>   |
     Then the set light schedule async response contains
       | DeviceIdentification | TEST1024000000001 |
+    And I wait 7 seconds
```

Comment:
This test is actually a out of scope of FLEX-5527. It concluded, based upon the occurence of a SOAP fault, that the schedule was split into several pages. Since that is stupid and faulty, I changed that, into a checking on the reception of a success message as a last step. I don't directly see a way (without investing quite some time) to get around the delay. During that time, the pages are sent to the device.

If you really wan't to get rid of the delay in tests, I would just @skip this test, since it did not do any good before.